From 2c6d4b82536ec029ade85a415fca49302899010c Mon Sep 17 00:00:00 2001
From: Laurent Pinchart <laurent.pinchart+renesas@ideasonboard.com>
Date: Wed, 19 Jun 2013 13:42:53 +0200
Subject: [PATCH 1002/1083] drm/rcar-du: Enable compilation on all ARM
 platforms

Signed-off-by: Laurent Pinchart <laurent.pinchart+renesas@ideasonboard.com>
(cherry picked from commit 32a34937372ef5ed1feee9ed9f8476201e3d2ffa)

Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit c3966076695700cf6999bdc74a75e9e5d283c69c)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 drivers/gpu/drm/rcar-du/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/rcar-du/Kconfig b/drivers/gpu/drm/rcar-du/Kconfig
index 2eb7d23..72887df 100644
--- a/drivers/gpu/drm/rcar-du/Kconfig
+++ b/drivers/gpu/drm/rcar-du/Kconfig
@@ -1,6 +1,6 @@
 config DRM_RCAR_DU
 	tristate "DRM Support for R-Car Display Unit"
-	depends on DRM && ARCH_SHMOBILE
+	depends on DRM && ARM
 	select DRM_KMS_HELPER
 	select DRM_KMS_CMA_HELPER
 	select DRM_GEM_CMA_HELPER
-- 
1.8.3.2

